static/js/tools/scatter/util.ts

```diff
@@ -246,6 +220,25 @@ export function applyHashBoolean(
   return val === "1";
 }
 
+/**
+ * Fetches a population point scale state from the given url query params
+ * @param params url query parameters
+ * @param key query parameter key containing population point scale state
+ * @returns PointScaleState value
+ */
+export function applyHashPopulation(
+  params: URLSearchParams,
+  key: string
```

Comment:
nit: don't need to pass in the key because will always use the same key (`FieldToAbbreviation.showPopulation`) to get the population from the hash

Reply:
done